tests/unit/api/main.py

```diff
@@ -132,6 +132,54 @@ def test_duplicate_project_create(self):
         with pytest.raises(api.DuplicateError):
             api.project_create('acme-corp')
 
+class TestProjectAddDeleteNetwork:
```

Comment:
what is a good way to not write them this way? the auth tests don't do this, but with the current structure of main.py I'm not sure how to make my few tests look like that without rewriting the rest of them